ModelicaTest/MultiBody.mo

```diff
@@ -8844,44 +8844,51 @@ This test model was proposed in ticket <a href=\"https://github.com/modelica/Mod
     model AbsoluteSensor "Simple pendulum with one revolute joint and one body"
       extends Modelica.Icons.Example;
       parameter Real tol=1e-3;
-      inner MultiBody.World world(gravityType=MultiBody.Types.GravityTypes.UniformGravity)
-        annotation (Placement(transformation(extent={{-80,0},{-60,20}})));
+      inner MultiBody.World world(
+        enableAnimation=false,
+        gravityType=MultiBody.Types.GravityTypes.UniformGravity)
+        annotation (Placement(transformation(extent={{-100,-10},{-80,10}})));
       MultiBody.Joints.Revolute rev(
-        n={0,0,1},
+        n(displayUnit="1") ={0,0,1},
         useAxisFlange=true,
         phi(fixed=true),
-        w(fixed=true)) annotation (Placement(transformation(extent={{-40,0},{-20,
-                20}})));
+        w(fixed=true)) annotation (Placement(transformation(extent={{-60,-10},{-40,10}})));
+      MultiBody.Joints.Revolute rev1(
+        n(displayUnit="1") = {1,0,0},
```

Comment:
(And similarly for this n).

Reply:
This was added to by Dymola 2023x. I didn't modify this.